test_integration/1_orchestrator_basic_config_suite_test.go

```diff
@@ -0,0 +1,137 @@
+package test_integration
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/suite"
+	"strings"
+	"testing"
+)
+
+type OrchestratorBasicConfigSuite struct {
+	BaseDockerComposeTestSuite
+}
+
+func NewOrchestratorBasicConfigSuite() *OrchestratorBasicConfigSuite {
+	s := &OrchestratorBasicConfigSuite{}
+	s.GlobalRunIdentifier = globalTestExecutionId
+	s.SuiteRunIdentifier = strings.ToLower(strings.Split(uuid.New().String(), "-")[0])
+	return s
+}
+
+func (s *OrchestratorBasicConfigSuite) SetupSuite() {
+	rawDockerComposeFilePath := "./common_assets/docker_compose_files/orchestrator-node-with-custom-start-command.yml"
+	s.Context, s.Cancel = context.WithCancel(context.Background())
+
+	orchestratorConfigFile := s.commonAssets("nodes_configs/1_basic_orchestrator_config.yaml")
+	orchestratorStartCommand := fmt.Sprintf("bacalhau serve --config=%s", orchestratorConfigFile)
+	extraRenderingData := map[string]interface{}{
+		"OrchestratorStartCommand": orchestratorStartCommand,
+	}
+	s.BaseDockerComposeTestSuite.SetupSuite(rawDockerComposeFilePath, extraRenderingData)
+
+	_, err := s.executeCommandInDefaultJumpbox(
+		[]string{
+			"mkdir",
+			"-p",
+			fmt.Sprintf("/app/%s", s.SuiteRunIdentifier),
+		})
+	s.Require().NoErrorf(err, "Error creating a suite tmp folder in the jumpbox node: %q", err)
+}
+
+func (s *OrchestratorBasicConfigSuite) TearDownSuite() {
+	s.T().Log("Tearing down [Test Suite] in OrchestratorBasicConfigSuite...")
+	s.BaseDockerComposeTestSuite.TearDownSuite()
+}
+
+func (s *OrchestratorBasicConfigSuite) TestNodesCanBeListed() {
+	nodeListOutput, err := s.executeCommandInDefaultJumpbox([]string{"bacalhau", "node", "list", "--output=json"})
+	s.Require().NoErrorf(err, "Error listing nodes: %q", err)
+
+	marshalledOutput, jsonType, err := s.unmarshalJSONString(nodeListOutput)
+	s.Require().NoErrorf(err, "Error unmarshalling json string: %q. JSON output received: %q", err, nodeListOutput)
+	s.Require().Equalf("array", jsonType, "incorrect json output type")
+
+	marshalledOutputArray := marshalledOutput.([]interface{})
+	s.Require().Equalf(1, len(marshalledOutputArray), "Node count should be 1")
+
+	nodeInfo := marshalledOutputArray[0].(map[string]interface{})["Info"].(map[string]interface{})
+
+	nodeType := nodeInfo["NodeType"].(string)
+	s.Require().Equal("Requester", nodeType)
+
+	nodeLabels := nodeInfo["Labels"].(map[string]interface{})
+	s.Require().Equal("label1Value", nodeLabels["label1"].(string))
+	s.Require().Equal("label2Value", nodeLabels["label2"].(string))
```

Comment:
should we have these as constants somewhere?

Reply:
I though about it, but decided to make it in a way that each test "suite" has its out test files, making tests not interconnected as much as possible.

And since each test will have its own test files with some test data in them, I think having the results explicitly stated in the tests will make it easier to read, and not affected by a common constant that is used by many tests.

Reply:
@frrist Correct, each test suite has an set of config files for it. They can be found in the SetupSuite function.